instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/sqsattributes_test.go

```diff
@@ -0,0 +1,195 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelaws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+	"github.com/aws/smithy-go/middleware"
+	"github.com/stretchr/testify/assert"
+
+	"go.opentelemetry.io/otel/attribute"
+)
+
+func TestSQSDeleteMessageBatchInput(t *testing.T) {
+	input := middleware.InitializeInput{
+		Parameters: &sqs.DeleteMessageBatchInput{
+			QueueUrl: aws.String("test-queue-url"),
+		},
+	}
+
+	attributes := SQSAttributeSetter(context.TODO(), input)
+
+	assert.Contains(t, attributes, attribute.String("messaging.url", "test-queue-url"))
```

Comment:
Same here. Consider using the semconv defined key so when name changes occur this doesn't break.

Reply:
Updated.